ci/process-pull-request-http.py

```diff
@@ -172,7 +172,7 @@ def action_check_permissions(self, git, pr, perms, tests):
     self.action_approval_required(git, pr, perms, tests)
 
   def action_approval_required(self, git, pr, perms, tests):
-    pull = git.get_pull(pr, cached=True)
```

Comment:
also here, I am not sure why you need to get the PR if then it's not needed. Does it update the cache?